### PR TITLE
Cleanup universities data

### PR DESCRIPTION
**Pull request by MdTehseenKhan**

### CODE REVIEW

**Review by MdTehseenKhan — COMMENTED** [review]
It was a duplicate record.

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -120248,18 +120248,6 @@
         ],
         "country": "France"
     },
-    {
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json:120251]: It was a duplicate record.

File: world_universities_and_domains.json [hunk 2]
```diff
@@ -120248,18 +120248,6 @@
...
-            "http://www.u-pec.fr/"
-        ],
-        "name": "Université Paris est Créteil Val de Marne Paris XII",
-        "alpha_two_code": "FR",
-        "state-province": null,
-        "domains": [
-            "u-pec.fr"
-        ],
-        "country": "France"
-    },
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json:120262]: It was a duplicate record.

File: world_universities_and_domains.json [hunk 3]
```diff
@@ -47490,7 +47490,7 @@
       "web_pages": [
           "https://www.iutripura.edu.in/"
       ],
-      "name": "ICFAI University",
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json:47493]: Added the campus state, as added in other campuses, so the name does not conflict with the head campus shown below:
```json
{
      "web_pages": [
          "http://www.icfai.org/"
      ],
      "name": "ICFAI University",
      "alpha_two_code": "IN",
      "state-province": null,
      "domains": [
          "icfai.org"
      ],
      "country": "India"
  },
 ```

File: world_universities_and_domains.json [hunk 4]
```diff
@@ -98403,18 +98403,6 @@
...
-          "http://www.bugemauniv.ac.ug/"
-      ],
-      "name": "Bugema University",
-      "alpha_two_code": "UG",
-      "state-province": null,
-      "domains": [
-          "bugemauniv.ac.ug"
-      ],
-      "country": "Uganda"
-  },
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json:98417]: It was a duplicate record but without the 'state-province' data.

File: world_universities_and_domains.json [hunk 5]
```diff
@@ -99351,18 +99351,6 @@
...
-          "http://www.univer.kharkov.ua/"
-      ],
-      "name": "Kharkiv National University",
-      "alpha_two_code": "UA",
-      "state-province": null,
-      "domains": [
-          "univer.kharkov.ua"
-      ],
-      "country": "Ukraine"
-  },
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json:99365]: Removed the old record and added the old domain and website to the new record. The old website is redirecting to the new one.

File: world_universities_and_domains.json [hunk 6]
```diff
@@ -101177,18 +101177,6 @@
...
-          "http://www.port.ac.uk/"
-      ],
-      "name": "University of Portsmouth",
-      "alpha_two_code": "GB",
-      "state-province": null,
-      "domains": [
-          "port.ac.uk"
-      ],
-      "country": "United Kingdom"
-  },
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json]: There were two different records for this university with different websites and domains. "Myport" is not a separate university, but rather an online portal or system used by the University of Portsmouth. It's likely short for "My Portsmouth". Myport is typically an intranet or web-based service provided by the University of Portsmouth for its students and staff.

File: world_universities_and_domains.json [hunk 7]
```diff
@@ -107818,18 +107818,6 @@
...
-          "http://www.dupage.edu"
-      ],
-      "name": "College of DuPage",
-      "alpha_two_code": "US",
-      "state-province": null,
-      "domains": [
-          "dupage.edu"
-      ],
-      "country": "United States"
-  },
```
Comment (MdTehseenKhan) [inline world_universities_and_domains.json:107832]: Expand down the changed file and you will find the next record with a working website that is http://www.cod.edu/.

**Comment by yigitguler** [comment]
Thank you!

**Pull request merged**